_includes/field-description.md

```diff
@@ -0,0 +1,59 @@
+## Description
+
+The `Description` field is a textual definition max 40 characters of the
+purchase. It is needed to specify what the purchase actually is for
+the consumer. Below you will find a abbreviated code example of Card Payments
+`Purchase`.
+As you can see the `description` field is set to be
+`"test purchase - orderNumber28749347"` in the the code example.
+The images below will show you the payment UI for the
+Redirect and Seamless View scenario. Notice that for Redirect, the description
+will be shown as `"test purchase - orderNumber28749347"`, as set in the code example.
+For the Seamless view scenario, the description is not shown in the payment
+window and therefore it doesnt need to be included.
```

Comment:
I think `description` is required for all requests, so it must be included regardless. I think that's why people wonder why it's required.

```suggestion
window, but it is still required in the initial request.
```